library/alloc/Cargo.toml

```diff
@@ -26,6 +26,11 @@ compiler-builtins-no-f16-f128 = ["compiler_builtins/no-f16-f128"]
 compiler-builtins-mangled-names = ["compiler_builtins/mangled-names"]
 # Make panics and failed asserts immediately abort without formatting any message
 panic_immediate_abort = ["core/panic_immediate_abort"]
+# Make the optimizer assume panics are unreachable.
+panic_unreachable_unchecked = [
```

Comment:
I propose to rename this feature to `unsafe_panic_unreachable_unchecked`, so that its name includes word `unsafe`, because this is actually an unsafe API (like `#[no_mangle]` attribute), and AFAIU all unsafe APIs tend to include `unsafe` in their usage (as part of their name, as a surrounding `unsafe` block, etc).

Alternatively, if this PR is a joke, I'd propose `crabs_never_panic` as a feature name instead.